bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/siteadmin/aem61/SiteadminPage.java

```diff
@@ -116,17 +115,14 @@ public SiteadminActions copyPage(String title, String destination) {
 
   @Override
   public SiteadminActions waitForPageCount(int pageCount) {
-    boolean conditionNotMet = !webElementUtils.isConditionMet(webDriver -> {
+    bobcatWait.verify(wd -> {
```

Comment:
could we still use `webDriver` instead of `wd` variable name? for junior develpopers... please..
(nice to have)